src/Modules/OrchardCore.Commerce/Views/ShoppingCartWidgetPart.cshtml

```diff
@@ -1,9 +1,5 @@
 @model ShoppingCartWidgetPartViewModel
 
-<style asp-src="~/OrchardCore.Commerce/css/shopping-cart-widget.min.css"></style>
+@Model.ItemCount
```

Comment:
The purpose of this is not really clear to me. I can't imagine a scenario where I'd want to customize just the text inside the circle. I could see wanting to customize the circle itself (e.g. adding a dropdown or hiding it). So I think the `<div class="shopping-cart-item-count text-white bg-danger text-center">` should be moved into this shape.